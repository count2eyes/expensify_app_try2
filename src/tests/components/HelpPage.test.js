import React from "react";
import { shallow } from "enzyme";
import Help from "../../components/HelpPage";

test("should render Help correctly", () => {
  const wrapper = shallow(<Help />);
  expect(wrapper).toMatchSnapshot();
});
