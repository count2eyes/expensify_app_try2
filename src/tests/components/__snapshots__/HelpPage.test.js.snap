// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Help correctly 1`] = `
<div>
  <h3>
    no help's here
  </h3>
</div>
`;
